### PR TITLE
Add ExDoc config in mix.exs and supporting file to generate the doc

### PR DESCRIPTION
**Pull request by fabriziosestito**
image

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+common mistakes and improve the quality of all contributions.
+
+## Opening issues
+
+Before creating a new issue make sure you use the search functionality to confirm
+that a similar issue doesn't already exist. Next, make sure you properly label
+the issue as per our [labels](https://github.com/trento-project/web/labels)
+
+If you are reporting a `bug`, please share a file generated using the
+`trento-support.sh` script with the following params:
```
Comment (nelsonkopliku) [inline CONTRIBUTING.md:16]: Is this `trento-support.sh` thing relevant for wanda?

**Review by nelsonkopliku — COMMENTED** [review]
Nice!

But be aware also of this PR https://github.com/trento-project/wanda/pull/80, some conflicts may raise. /cc @rtorrero 

In relation to the TOC, I believe removing it is a smart move :rocket: 
- ExDoc builds the navigation, as you mentioned
- github itself does that
- solves this discussion https://github.com/trento-project/wanda/pull/80#discussion_r1023027448

Good one!

**Review by dottorblaster — COMMENTED** [review]
LGTM

**Comment by rtorrero** [comment]
> Nice!
> 
> But be aware also of this PR #80, some conflicts may raise. /cc @rtorrero
> 
> In relation to the TOC, I believe removing it is a smart move rocket
> 
>     * ExDoc builds the navigation, as you mentioned
> 
>     * github itself does that
> 
>     * solves this discussion [#80 (comment)](https://github.com/trento-project/wanda/pull/80#discussion_r1023027448)
> 
> 
> Good one!

Merge this first, I'll rebase and change what isn't covered by this :+1:

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+# How to contribute
```
Comment (arbulu89) [inline CONTRIBUTING.md:1]: We could maybe upload this file to our new `docs` repo? 
It looks something generic enough to have it as template

File: guides/expression_language.md [hunk 3]
```diff
@@ -163,14 +154,15 @@ let last_element = some_list[-1];
...
-| `get`    | position, counting from end if < 0 | gets a copy of the element at a certain position (`()` if the position is not valid) |
-| `len`    | _none_                             | returns the number of elements |
-| `filter` | predicate (usually a closure) | constructs a new array with all items that return `true` when called with the predicate function taking the following parameters:<ol><li>array item</li><li>_(optional)_ offset position</li></ol> |
-| `all`    | predicate (usually a closure) | returns `true` if all items return `true` when called with the predicate function taking the following parameters:<ol><li>array item</li><li>_(optional)_ offset position</li></ol>  |
+| Function | Parameter(s)                       | Description                                                                                                                                                                                        |
+| -------- | ---------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```
Comment (arbulu89) [inline guides/expression_language.md:158]: Damn tables in md!!
Cannot they make something that doesn't require so many dashes XD

File: mix.exs [hunk 4]
```diff
@@ -22,6 +27,55 @@ defmodule Wanda.MixProject do
...
+    [
+      main: "readme",
+      logo: "priv/static/images/trento.svg",
+      extra_section: "GUIDES",
+      source_url: @source_url,
+      assets: "guides/assets/",
+      extras: extras(),
+      groups_for_extras: groups_for_extras(),
+      groups_for_modules: [
+        Domain: [~r/Trento.Domain.*/],
```
Comment (arbulu89) [inline mix.exs:40]: This `groups_for_modules` is wrong hehe
Need to be updated to wanda

**Review by arbulu89 — CHANGES_REQUESTED** [review]
So far so good!
There is one major issue in the `groups_for_modules` entry in the mix file.
Green light besides that

File: mix.exs (hunk 4 above)
Comment (fabriziosestito) [reply mix.exs]: updated

**Review by dottorblaster — APPROVED** [review]
LGTM

File: mix.exs [hunk 5]
```diff
@@ -22,6 +27,63 @@ defmodule Wanda.MixProject do
...
+      "CHANGELOG.md",
+      "CONTRIBUTING.md",
+      "guides/specification.md",
+      "guides/expression_language.md",
+      "guides/gatherers.md"
+    ]
+  end
+
+  defp groups_for_extras do
+    %{
```
Comment (arbulu89) [inline mix.exs:61]: It looks strange that we use a map here and a list of keywords in the `groups_for_modules` XD (i see the 2nd in most of the projects)

**Review by arbulu89 — APPROVED** [review]
Great!
Ready to merge!!

**Pull request merged**